test/MOF/jump_integration.jl

```diff
@@ -0,0 +1,51 @@
+import Pkg
+Pkg.add("Ipopt")
+
+using JuMP, MathOptFormat, Ipopt
+
+function MOI.copy_to(dest, src::Model; kwargs...)
+    if src.nlp_data !== nothing
+        MOI.set(src, MOI.NLPBlock(), JuMP._create_nlp_block_data(src))
+    end
+    MOI.copy_to(dest, backend(src); kwargs...)
+end
```

Comment:
So the question is, should these `copy_to` methods be included in JuMP? Or should they sit in MathOptFormat?